src/app/public/modules/vertical-tabset/vertical-tabset.component.scss

```diff
@@ -21,6 +22,11 @@
   .sky-vertical-tabset-group-container {
     @include sky-border(dark, right);
   }
+  @include sky-theme-modern {
```

Comment:
Have we verified this style working? I could be wrong but I wouldn't have expected this style to work in this order. I would have thought this inner style would have had to go in the `@include sky-host-responsive-container-sm-min()` below (currently on line 63) due to the ordering of the css classes on the page.

Reply:
Yes. it was working, but you're right it should probably be at bottom to make it consistent that modern styles are down there together. I moved it and confirmed it worked. Also found I needed to add this to section form scss as it wasn't getting this from this vertical tab declaration. thx.